src/aiodynamo/utils.py

```diff
@@ -197,3 +201,43 @@ def deparametetrize(
     for key, value in params.names.items():
         expression = expression.replace(key, value)
     return expression
+
+
+@dataclass(frozen=True)
+class MinLen2AppendOnlyList(Generic[T]):
```

Comment:
ideas:
`MinTwoItemAppendOnlyList`
`MinTwoAppendOnlyList`
`AtLeastTwoAppendOnlyList`
`TwoOrMoreAppendOnlyList`
...

Reply:
How about "Multiple" or "Several"?

On another note, this is more of an immutable list, calling it "append only" feels misleading, as the stored list is never modified after creation.

So maybe a tuple then?

How about "multivariate tuple" which is a thing in statistics, a tuple containing 2 or more elements.

Reply:
yea it's no longer a list. it's more of a tuple-like iterable with a minimum length